src/classes/UTIL_UnitTestData_TEST.cls

```diff
@@ -523,6 +523,49 @@ public class UTIL_UnitTestData_TEST {
         return mappingService;
     }
 
+    /**
+     * @description Utility method to create Data Import Records in a Gift Batch
+     * @return {dataImports} List of Data Import records
+    */
+    public static List<DataImport__c> createDIRecordsInABatch (Integer numberOfRecords) {
```

Comment:
Because this is specific to new-GE Batch DIs (batch version=2) and not old-style BGE records (batch version < 2), I think this should be 'createDIRecordsInAGEBatch' or something to indicate that these records represent that specific setup